Sources/TokamakCore/CGStubs.swift

```diff
@@ -15,7 +15,11 @@
 //  Created by Max Desiatov on 08/04/2020.
 //
 
+#if os(WASI)
 import Glibc
+#else
+import Foundation
+#endif
```

Comment:
Sorry about changing my mind, here's what would be most cross-platform in my opinion:

```suggestion
#if canImport(Glibc)
import Glibc
#elseif canImport(Darwin)
import Darwin
#endif
```

Reply:
No worries!